app-backup/rdup/rdup-1.1.15-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Generate a file list suitable for full or incremental backups"
+HOMEPAGE="https://github.com/miekg/rdup/releases"
+SRC_URI="https://github.com/miekg/rdup/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-arch/libarchive
+	dev-libs/glib:2
+	dev-libs/libpcre
+	dev-libs/nettle
+"
+DEPEND="${RDEPEND}"
+BDEPEND="test? ( dev-util/dejagnu )"
+
+src_prepare() {
+	default
+	sed -i -e 's/ -Werror//' GNUmakefile.in || die "Failed to fix Makefile"
+	eautoreconf
+}
+
+src_configure() {
+	econf $(use_enable debug)
+}
+
+src_test() {
+	if use debug; then
+		ewarn "Test phase skipped, as it is known to fail with USE=\"debug\"."
```

Comment:
should really do REQUIRED_USE="test? ( !debug )" instead of this, we don't want it to "silently" skip if someone wanted tests